tools/add_font.py

```diff
@@ -23,6 +55,30 @@
                      ' for a -ext subset.')
 
 
+def family_url(name):
+  url_prefix = 'http://fonts.google.com/specimen/'
+  dl_name = name.replace(' ', '+')
+  return url_prefix + dl_name
+
+
+def family_exists(name):
+  """Check if a font family exists on fonts.google.com"""
+  url = family_url(name)
+  request = requests.get(url)
+  if request.status_code == 200:
+      return True
+  return False
+
+
+def _GetFamilyMetaData(name):
+  """Get family information from the frontend GF api"""
+  api_prefix = 'http://fonts.google.com/metadata/fonts/'
```

Comment:
I chatted to @rsheeter offline to confirm that this PR is part of such embarkation by @m4rc1e :)